css/styles.css

```diff
@@ -0,0 +1,123 @@
+.background-image {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: -1;
+  background: linear-gradient(rgb(25, 25, 50), rgba(25, 25, 50, 0.5)), linear-gradient(rgba(25, 25, 50, 0.5), rgb(25, 25, 50)), url('../images/final-background-image.png');
+  background-position: center;
+  background-repeat: repeat-x;
+  object-fit: fill;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.justify-right {
+  justify-content: right;
+}
+
+.row-nav-full {
+  margin-top: 10px;
+}
+
+.column-nav-full {
+  width: 90%;
+  margin: auto;
+}
+
+.home-content {
+  position: relative;
+  top: 180px;
```

Comment:
This is a **_big_** movement. Is this just to create space around the element? You should use padding or margin if that is the case.